spec/unit/actions/collect_dynamic_schedule_params_spec.rb

```diff
@@ -52,5 +51,16 @@
       }
       expect(result[:params][:schedule_params]).to eq(expected)
     end
+
+    it 'does not print the one with param_secure' do
+      result = subject.class.call(params)
```

Comment:
We use the `GoodData::LCM2.run_action` method for testing actions now.

Reply:
Fixed.